### PR TITLE
Set CKKSRNS_SCHEME as schemeTag in CKKSPackedEncoding constructors

### PR DESCRIPTION
**Pull request by dsuponitskiy**

### CODE REVIEW

File: src/pke/include/encoding/ckkspackedencoding.h [hunk 1]
```diff
@@ -143,7 +143,7 @@ class CKKSPackedEncoding : public PlaintextImpl {
     bool Decode() {
         OPENFHE_THROW(
             not_available_error,
-            "CKKSPackedEncoding::Decode() is not implemented. Use CKKSPackedEncoding::Decode(noiseScaleDeg,scalingFactor,scalTech) instead.");
+            "CKKSPackedEncoding::Decode() is not implemented. Use CKKSPackedEncoding::Decode(...) instead.");
```
Comment (sarojaduality) [inline src/pke/include/encoding/ckkspackedencoding.h:146]: Why this change?

**Review by sarojaduality — COMMENTED** [review]
Looks good other than the error message change.

File: src/pke/include/encoding/ckkspackedencoding.h (hunk 1 above)
Comment (dsuponitskiy) [reply src/pke/include/encoding/ckkspackedencoding.h:146]: the comment is already outdated as bool Decode(size_t depth, double scalingFactor, ScalingTechnique scalTech, ExecutionMode executionMode) has got more arguments (4 instead of 3) now. also it is not a good practice to mention ALL function arguments in a different place as the chance that the comment will get obsolete gets higher over time.

Comment (sarojaduality) [reply src/pke/include/encoding/ckkspackedencoding.h:146]: Got it, makes sense
